packages/hpe_aruba_cx/data_stream/log/fields/fields.yml

```
@@ -0,0 +1,895 @@
+          description: >
+
+        - name: invalid_ip
+          type: ip
+          description: >
+
+        - name: ip_version
+          type: keyword
+          description: >
+
+        - name: local_diag
+          type: keyword
+          description: >
+
+        - name: local_state
+          type: keyword
+          description: >
+
+        - name: op_mode
+          type: keyword
+          description: >
+
+        - name: remote_diag
+          type: keyword
+          description: >
+
+        - name: remote_state
+          type: keyword
+          description: >
+
+        - name: requested_interval
+          type: keyword
+          description: >
+
+    - name: bgp
+      type: group
+      fields:
+        - name: id
+          type: keyword
+          description: >
+
+        - name: pg_name
+          type: keyword
+          description: >
+
+        - name: threshold_limit
+          type: long
+          description: >
+
+        - name: vtep_ip
+          type: ip
+          description: >
+
+    - name: cm
+      type: group
+      fields:
+        - name: cert_name
+          type: keyword
+          description: >
+
+        - name: days
+          type: long
+          description: >
+
+        - name: est_name
+          type: keyword
+          description: >
+
+        - name: profile_name
+          type: keyword
+          description: >
+
+    - name: config
+      type: group
+      fields:
+        - name: type
+          type: keyword
+          description: >
+
+        - name: value
+          type: keyword
+          description: >
+
+    - name: copp
+      type: group
+      fields:
+        - name: class
+          type: keyword
+          description: >
+
+    - name: cpu_rx
+      type: group
+      fields:
+        - name: filter_description
+          type: keyword
+          description: >
+
+    - name: dhcp
+      type: group
+      fields:
+        - name: config
+          type: keyword
+          description: >
+
+        - name: ipv6_address
+          type: keyword
+          description: >
+
+        - name: lease_ip_address
+          type: ip
+          description: >
+
+        - name: new_port
+          type: long
+          description: >
+
+        - name: server_ip_address
+          type: ip
+          description: >
+
+        - name: source_mac
+          type: keyword
+          description: >
+
+        - name: volume_name
+          type: keyword
+          description: >
+
+    - name: dns
+      type: group
+      fields:
+        - name: type
+          type: keyword
+          description: >
+
+    - name: dpse
+      type: group
+      fields:
+        - name: linecard_name
+          type: keyword
+          description: >
+
+    - name: ecmp
+      type: group
+      fields:
+        - name: route
+          type: keyword
+          description: >
+
+    - name: erps
+      type: group
+      fields:
+        - name: ring_id
+          type: long
```

```suggestion
          type: keyword
```

IDs should be keywords